docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,374 @@
+   - Yes: Access to artifacts are queried and pulled from their repository using the existing access controls.
+1. As a tool writer, I want to identify whether a registry supports reference types or not.
+   - Yes: This proposal doesn't require changes to the registry.
+1. As a tool writer, I would like the option to perform a server side blob mount when copying a large artifact between repositories.
+   - Yes: Artifacts can be copied with existing APIs.
+1. As a tool writer, I want to be able to include reference types within the Image Layout filesystem format.
+   - Yes: Artifacts may be added to the Image Layout in addition to the referenced manifests.
+
+### Content Management
+
+1. As a user, I want to store one or more artifacts in a registry with a reference to another related artifact.
+   - Yes: References are created using the unique tag syntax.
+1. As a user, when I delete an artifact, I want the option to delete one or more artifacts referencing it.
+   - Partial: Clients that delete a manifest should also delete tags with a digest referencing that manifest.
+     A client-side GC could also be created to prune digest tags for digests that are no longer available in that repository.
+     Client-side GC should not be performed when the user wants to push artifacts before the referenced image, or when creating references to images that are not stored in this repository.
+1. As a user, I want to push an artifact that references another artifact that doesn't exist on the registry yet.
+   - Yes: Digest tags can be created that point to manifests that do not exist yet.
+     Client-side GC should not be performed in these environments.
+1. As a user, I want to move an artifact and one or more artifacts referencing it from one registry to another.
+   - Yes: Artifacts may be copied between registries.
+1. As a registry operator, I want to help users understand how they can manage the lifecycle of their artifacts.
+   - Partial: Lifecycle management depends on client-side processing to remove tags to artifacts that should be deleted.
+1. As a registry operator, I want to allow users to "lock" the tags to their artifacts.
+   - Yes: Tag locking can be used with the unique tags to extend existing manifests with new artifact references.
+     There is no need to mutate existing tags, and the originator can package their index in advance to push once with the images and all reference artifacts together.
+1. As an artifact producer, I want to update an existing artifact with a newer artifact.
+   - Yes: New artifacts are associated by pushing the artifact and the unique tag referencing the target manifest.
+1. As an artifact producer, I want to push multiple artifacts concurrently (possibly from separate systems) without encountering race conditions or other errors.
+   - Yes: Each artifact may be pushed separately with a unique artifact manifest and tag that references the target manifest.
```

Comment:
Weren't we getting away from unique tags with this proposal? If multiple tags exist, do all manifests need to be pulled, and how does the client know which one is the newest?

Reply:
I don't think that I understand the use case here. Is it intended to tease out that index modifications are racy?

Reply:
Sorry, I should have clarified, multiple digest tags. I thought you were pushing for a single predictable digest tag, and not a different digest tag for each version of the extended index.

Reply:
Sorry, indeed, it's a copy/paste I should have resolved 😅 
Changing to:
`No: If two manifests with the same digest are pushed to the same registry path, a race condition will occur. However, this problem exists in current registries.`

Reply:
Fixed https://github.com/opencontainers/wg-reference-types/pull/52/commits/47a0daa6a00ac31c25ba0d180c158ff9442e6634